packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -283,26 +284,14 @@ export const getSyncs = async (nangoConnection: Connection): Promise<(Sync & { s
             'models'
         );
 
-    // console.log(q.toQuery());
     const result = await q;
     const syncsWithSchedule = result.map(async (sync) => {
         const { schedule_id } = sync;
-        const schedule = scheduleResponse?.schedules.find((schedule) => schedule.scheduleId === schedule_id);
-        // if a disagreement trust temporal
-        if (schedule?.info?.paused && sync.schedule_status !== SyncStatus.PAUSED) {
-            sync = {
-                ...sync,
-                schedule_status: SyncStatus.PAUSED
-            };
-            await updateScheduleStatus(schedule_id, SyncCommand.PAUSE, null, nangoConnection.environment_id);
-        } else if (!schedule?.info?.paused && sync.schedule_status === SyncStatus.PAUSED) {
-            sync = {
-                ...sync,
-                schedule_status: SyncStatus.RUNNING
-            };
-            await updateScheduleStatus(schedule_id, SyncCommand.UNPAUSE, null, nangoConnection.environment_id);
+        const schedule = await syncClient?.describeSchedule(schedule_id);
+        let futureActionTimes: number[] = [];
+        if (schedule) {
+            futureActionTimes = schedule?.info?.futureActionTimes?.map((long) => long.seconds?.toNumber()) as number[];
```

Comment:
would the value of `futureActionTimes` be `undefined` if `.info` or `.futureActionTimes` is undefined?

Reply:
Updated this to check for these properties.

Reply:
you can just default to `|| []` so you don't have to check

Reply:
Linter complains when I try to update to this, going to leave as is for now -- unless you have any issues